lib/prompts/checkbox.js

```diff
@@ -86,7 +90,8 @@ Prompt.prototype.render = function (error) {
   var bottomContent = '';
 
   if (this.firstRender) {
-    message += '(Press <space> to select)';
+    message += '(Press <space> to select, <a> to select all,\n<n> to select none,';
```

Comment:
The `a` key can do both. It select all if not all are selected, and unselect all if all are selected.